### ISSUE TITLE
Unable to highlight text

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#582]
### Expected behavior
We should be able to highlight text

### Actual behavior
Unable to highlight text

### Steps to reproduce

Version: 0.8.0 (Chromium:76.0.3803.0)
Type .api for help and .exit to quit

> openBrowser()
 ✔ Browser opened
> goto("github.com")
 ✔ Navigated to url http://github.com
> highlight("Sign in")
 ✔ Highlighted the element matching text "Sign in"

### Versions
```
➜  issue535 taiko -v
Version: 0.8.0 (Chromium: 76.0.3803.0) 574e989dc09432ddcf4494e44257d63da4c8bf53
```

**Comment by NivedhaSenthil** [comment]
Fix available in b04dbbf

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the commit version ```b04dbbf ```
